Makefile

```diff
@@ -1,5 +1,13 @@
-NAME ?= summerwind/actions-runner-controller
+ifdef DOCKER_USER
+	NAME ?= ${DOCKER_USER}/actions-runner-controller
+else
+	NAME ?= summerwind/actions-runner-controller
+endif
+DOCKER_USER ?= $(shell echo ${NAME} | cut -d / -f1)
+RUNNER_NAME ?= ${DOCKER_USER}/actions-runner
 VERSION ?= latest
+TEST_REPO ?= ${DOCKER_USER}/actions-runner-controller
```

Comment:
I think this should usually be another github repository for testing various github actions workflows and jobs, rather than a relatively big repo containing all the actions-runner-controller code.